index.html

```diff
@@ -28,7 +28,13 @@
   <header>
     <a href="#home" class="logo">Ezeoke Stanley</a>
     <nav>
-         <img src="./icons/menu.png" alt="menu" class="envelope">
+      <ul>
+        <li><img src="./icons/menu.png" alt="menu" class="envelope"></li>
+        <li><a href="#portfolio" class="desktop-icon">Portfolio</a></li>
+        <li><a href="#about" class="desktop-icon">About</a></li>
+        <li><a href="#contact" class="desktop-icon">Contact</a></li>
+        <li><img src="./images/envlope-desktop.png" alt="envelope icon" class="desktop-icon"></li>
+       </ul>
      </nav>
 </header>
          <!--Header-->
```

Comment:
- [ ] Please you have a broken icon image, fix this and make your design closed to the original one